eclass/qt5-build.eclass

```diff
@@ -114,7 +114,7 @@ BDEPEND="
 	virtual/pkgconfig
 "
 if [[ ${PN} != qttest ]]; then
-	DEPEND+=" test? ( ~dev-qt/qttest-${PV} )"
+	DEPEND+=" test? ( ~dev-qt/qttest-$(ver_cut 1-3) )"
```

Comment:
I guess another option would be `>=dev-qt/qttest-5.15.2:5` as well here.

Reply:
yeah you can go with `>=5.15.2` but either way is fine with me